staging/src/k8s.io/apimachinery/pkg/apimachinery/announced/group_factory.go

```diff
@@ -42,10 +42,16 @@ type GroupVersionFactoryArgs struct {
 
 // GroupMetaFactoryArgs contains the group-level args of a GroupMetaFactory.
 type GroupMetaFactoryArgs struct {
+	// GroupName is the name of the API-Group
+	//
+	// example: 'servicecatalog.k8s.io'
 	GroupName              string
 	VersionPreferenceOrder []string
-	ImportPrefix           string
-
+	// ImportPrefix is the base go package of the API-Group
```

Comment:
> It always looks like a go package, but I do not know why.

There is/was some kind of reflective go-type lookup to build a hard coded RESTMapper.